Program.cs

```diff
@@ -1,2 +1,88 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World! This is simple inventory management system");
+﻿using SimpleInventoryManagementSystem.ProductsManegements;
+
+Inventory Inv=new Inventory();
+PrintWelcome();
+ShowMainMenu();
+
+
+void PrintWelcome()
+{
+    Console.ForegroundColor = ConsoleColor.Yellow;
+    Console.WriteLine(@"
+*********************************************************
+*                                                       *
+*        Welcome to Inventory Management System         * 
+*                                                       *
+*********************************************************
+    ");
+    Console.ResetColor();
+    Console.WriteLine("Press any key to start!");
+    Console.ReadLine();
+    Console.Clear();
+
+}
+
+void ShowMainMenu()
+{
+    string? choice;
+    while (true) { 
+        Console.ForegroundColor = ConsoleColor.White;
+        Console.ResetColor();
+        Console.Clear();
+
+        Console.WriteLine($"\nYou currently have {Inventory.products.Count} product");
+
+        Console.WriteLine(@"
+--------------------------
+Select an action to statrt ->
+--------------------------
+");
+
+        Console.WriteLine("1.Add product");
+        Console.WriteLine("2.View all products ");
+        Console.WriteLine("3.Edit product");
+        Console.WriteLine("4.Delete product");
+        Console.WriteLine("5.Search product");
+        Console.WriteLine("0.Exit");
```

Comment:
Based on the single responsibility can we add these lines in a separate method for example  `MainMenuOption`